### PR TITLE
Perf tests in development branch

### PR DESCRIPTION
**Pull request by madflow**
* This PR incorporates the perf-tests in the development branch (future master)
* I searched for "Perf" in the test file names. I hope I got them all.
* They only took ~10 minutes on my laptop. 
* It was not possible to exclude a group in a specific testsuite in PHPunit. You can only exclude groups "globally" in one config file. So there is no "run all tests" anymore. You will have to run them separately.

#616

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @madflow has signed the CLA. Thanks for the pull request!

File: README.md [hunk 1]
```diff
@@ -27,13 +27,10 @@ Full documentation can be found at [http://opensource.box.com/spout/](http://ope
...
+If you just want to check that everything is working as expected, executing the unit and functional tests is enough.
 
-If you want to run performance tests, you will need to checkout the `perf-tests` branch. Multiple test suites can then be run, depending on the expected output:
-
-* `phpunit` - runs the whole test suite (unit + functional + performance tests)
-* `phpunit --exclude-group perf-tests` - only runs the unit and functional tests
+* `phpunit` - runs unit and functional tests
 * `phpunit --group perf-tests` - only runs the performance tests
 
 For information, the performance tests take about 30 minutes to run (processing 1 million rows files is not a quick thing).
```
Comment (adrilo) [inline README.md]: can you update 30min to 10min?

Comment (madflow) [reply README.md]: done and rebased.

**Comment by madflow** [comment]
To merge or not to merge. This is the question.

**Comment by adrilo** [comment]
Merge!

**Pull request merged**